### PR TITLE
Feature#22 add damage records

### PR DESCRIPTION
**Pull request by Zuack55**
### What?
-  creation of the reporting functionality
- See Initializing the React Native project #41  for more information
### Why
- for the user to create damage reports
- See Initializing the React Native project #41 for more information.
## Testing
- Since the project is just initialized you only need to install the basic packages to visualize the application
```npm install```
- then you can initialize with an android emulator or via web
```npm start```

### CODE REVIEW

**Review by McFly1208 — APPROVED** [review]

**Review by arthurchina92 — APPROVED** [review]

File: src/views/LoginScreen.jsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class LoginScreen extends Component {
       this.state.inputValuePassword
     )
       .then((res) => {
+        console.log(res.user);
```
Comment (IvanGodinez21) [inline src/views/LoginScreen.jsx:36]: Remove this `console.log`

**Review by IvanGodinez21 — APPROVED** [review]

**Review by Matarratas777 — APPROVED** [review]
LGTM 🍆

File: src/views/AddRecord.jsx [hunk 2]
```diff
@@ -8,17 +8,84 @@ import { MaterialIcons } from "@expo/vector-icons";
...
+      image: null,
+      isLoading: false,
+      uploading: false,
+      descripcion: '',
+      markerLocation: {
+                latitude: 19.244,
+                longitude: -103.725
+      }
+    };
+  }
```
Comment (yovanha) [inline src/views/AddRecord.jsx:34]: Check the indentation of this file please

File: src/views/AddRecord.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { Text, View, ScrollView, TouchableOpacity } from "react-native";
+import { Text, ScrollView, TouchableOpacity, Button, Image, View, Platform, ActivityIndicator, Alert} from "react-native";
```
Comment (yovanha) [inline src/views/AddRecord.jsx:2]: Did you use a code formatter for this?

File: src/views/AddRecord.jsx [hunk 4]
```diff
@@ -8,17 +8,84 @@ import { MaterialIcons } from "@expo/vector-icons";
...
+    
+    const reporte = {
+      descripcion: this.state.descripcion,
+      idImg: id,
+      locacion: this.state.markerLocation,
+      fecha: new Date(),
+    };
+    
+      const newReporte = doc(collection(db, `users/${auth.currentUser.uid}/reports`));
+      const q = query(collection(db, 'users'));
```
Comment (yovanha) [inline src/views/AddRecord.jsx:65]: Could you please give a better name for this const?

**Review by yovanha — CHANGES_REQUESTED** [review]

File: src/views/AddRecord.jsx (hunk 2 above)
Comment (Zuack55) [reply src/views/AddRecord.jsx:34]: done

File: src/views/AddRecord.jsx (hunk 3 above)
Comment (Zuack55) [reply src/views/AddRecord.jsx:2]: yes but i forgot to apply it i am using prettier

File: src/views/AddRecord.jsx (hunk 4 above)
Comment (Zuack55) [reply src/views/AddRecord.jsx:65]: done

**Review by yovanha — APPROVED** [review]
LGTM!

**Pull request merged**